tests/ui/ReportActionComposeTest.tsx

```diff
@@ -228,6 +228,14 @@ describe('ReportActionCompose Integration Tests', () => {
     });
 
     describe('Message validation', () => {
+        beforeEach(() => {
+            jest.useFakeTimers();
+        });
+
+        afterEach(() => {
+            jest.useRealTimers();
+        });
```

Comment:
Why do we need this now?

Reply:
`scheduleOnUI` mock uses `setTimeout` (contrary to the mock that we had for `react-native-animated` counterpart), so without using 

```ts
act(() => {
                jest.advanceTimersByTime(1);
            });
```

the test will fail